torch_xla/experimental/custom_kernel.py

```diff
@@ -124,10 +124,17 @@ def wrapped_kernel(kernel: Callable,
         kernel, static_argnames=static_argnames).lower(*jax_args,
                                                        **kwargs).compiler_ir()
     payload = _extract_backend_config(ir)
-    output_shape, output_dtype = output_shape_dtype_fn(*args)
-    output = torch.empty(output_shape, dtype=output_dtype).to(xm.xla_device())
-    torch_xla._XLAC._xla_tpu_custom_call_(output, args, payload)
-    return output
+    # TODO: We can consider supporting un-array output.
+    outputs = []
+    for output_shape, output_dtype in output_shape_dtype_fn(*args):
```

Comment:
do we need to check `output_shape_dtype_fn ` returns an array of tuple of this works for a single tuple as well?